.github/workflows/docker.yml

```diff
@@ -0,0 +1,38 @@
+name: docker
+
+on:
+  push:
+    tags-ignore:
+      - "helm-*"
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Fetch all tags
+        run: git fetch --force --tags
+      -
+        name: get version tag
+        run: echo "TAG_VERSION=$(echo ${GITHUB_REF#refs/*/})" >> $GITHUB_ENV
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
why do we need to setup QEMU? we are only compiling for x86_64, are we?

Reply:
true, I've removed it now